packages/antd/src/widgets/SubmitButton/SubmitButton.js

```diff
@@ -5,7 +5,7 @@ const { getSubmitButtonOptions } = utils;
 export default ({ uiSchema }) => {
   const { submitText, norender, props: submitButtonProps }= getSubmitButtonOptions(uiSchema);
   if (norender) {return null;}
-  return (<Button  type="submit" {...submitButtonProps}>
+  return (<Button  htmlType={"submit"} {...submitButtonProps}>
```

Comment:
Consider simplifying this to `htmlType="submit"`

Reply:
Also I have done this.